src/liballoc_system/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Never update the start year for copyrights. If you really feel compelled to update the copyright then update the end year (or add an end year if it doesn't have one already) so `2015 - 2017`